### PR TITLE
TxPool config fix

### PR DESCRIPTION
**Pull request by Voxelot**
There was a missing link while implementing #307 to actually forward the config values from fuel-core to the fuel-txpool module, preventing the node from properly using the min_gas_fee and min_byte_fee settings. This PR finishes the piping of those config values and updates the integ tests to ensure it's working end-to-end.

### CODE REVIEW

File: fuel-tests/Cargo.toml [hunk 1]
```diff
@@ -23,6 +23,7 @@ fuel-crypto = { version = "0.4", features = ["random"] }
 fuel-gql-client = { path = "../fuel-client", features = ["test-helpers"] }
 fuel-storage = "0.1"
 fuel-tx = { version = "0.9", features = ["serde-types", "builder", "internals"] }
+fuel-txpool = { path = "../fuel-txpool" }
```
Comment (adlerjohn) [inline fuel-tests/Cargo.toml:26]: This needs a version number, no?

Comment (Voxelot) [reply fuel-tests/Cargo.toml:26]: this is a local unpublished crate, so it doesn't need version numbers for any deps coming from this repo

Comment (adlerjohn) [reply fuel-tests/Cargo.toml:26]: Ah right.

**Review by adlerjohn — COMMENTED** [review]
Should you add a failing test?

**Review by rakita — DISMISSED** [review]
lgtm

**Review by ControlCplusControlV — DISMISSED** [review]
LGTM

**Comment by Voxelot** [comment]
@adlerjohn added a failing test

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**